packages/client/src/runtime/utils/common.ts

```diff
@@ -400,6 +415,11 @@ export function isGroupByOutputName(type: string): boolean {
 
 export function isSchemaEnum(type: any): type is DMMF.SchemaEnum {
   return (
-    typeof type === 'object' && type.name && typeof type.name === 'string' && type.values && Array.isArray(type.values)
+    typeof type === 'object' &&
+    type !== null &&
+    type.name &&
+    typeof type.name === 'string' &&
+    type.values &&
+    Array.isArray(type.values)
```

Comment:
```suggestion
    typeof type === 'object' &&
    type !== null &&
    typeof type.name === 'string' &&
    Array.isArray(type.values)
```

I know it's an old code that just got reformatted, but I've noticed that couple of checks are already covered by other conditions.